google_cloud_run_deploy/values.py

```diff
@@ -38,6 +38,8 @@ def generate_terraform_tfvars_file(self, file_path):
         for param_name, param_value in self.items():
             if isinstance(param_value, (dict, list)):
                 write_value = json.dumps(param_value)
+            elif isinstance(param_value, bool):
+                write_value = str(param_value).lower()
```

Comment:
This seems unrelated to the `env` config option?

Reply:
it is but this fails otherwise